evil-common.el

```diff
@@ -3977,6 +3977,22 @@ PROPERTIES is a property-list which supports the following properties:
          (evil-motion-state))
        (switch-to-buffer-other-window buf))))
 
+;;; Window
+
+(defun evil-window-visible-height (&optional window)
+  "The visible height of WINDOW in lines.
+
+If no WINDOW is specified, use the selected one."
+  (interactive)
```

Comment:
why interactive?

Reply:
Oh, didn't mean to make it interactive. Removed!